idc_index/index.py

```diff
@@ -603,8 +603,9 @@ def _validate_update_manifest_and_get_download_size(
                 seriesInstanceuid,
                 s3_url,
                 series_size_MB,
-                index_crdc_series_uuid==manifest_crdc_series_uuid AS crdc_series_uuid_match,
+                index_crdc_series_uuid is not NULL as crdc_series_uuid_match,
```

Comment:
This has no impact imo, as if there is no match, the result is false before and after this change. but I'm fine either way.

Reply:
The point is that the result wasn't False before this change. The nature of LEFT JOIN is to have NULL for the items that are not matched. Comparison of anything with NULL will result in NULL, see https://cloud.google.com/bigquery/docs/reference/standard-sql/operators, which propagates into the pandas comparison later that was not working as intended. That following check was never tested properly, I am afraid.

Reply:
Thanks I did not know about the behavior of comparison with nulls. 

As for the accuracy of validation, we were indeed checking if every value is True, so the validation still should have been working as expected.